src/game/components/Dialogue.js

```diff
@@ -44,38 +44,27 @@ class Dialogue extends TextBox {
         this.displayDialogue();
     }
 
-    displayDialogue(isAnimated) {
+    displayDialogue() {
         this.hideDialogue();
         const { name, text } = this.dialogue[this.currentIndex];
-        const dialogue = `${name}:\n ${text}`;
-        if (isAnimated) {
-            super.displayDialogue(dialogue);
-        } else {
-            super.displayStaticDialogue(dialogue);
-        }
+        super.displayDialogue(`${name}:\n ${text}`);
         this.forwardButton.setVisible(this.currentIndex < this.dialogue.length);
         this.backButton.setVisible(this.currentIndex > 0);
     }
 
     advance() {
-        if (this.typingTimer?.loop) {
```

Comment:
Why was this removed? I had added this so that they could skip the typing animation and still read the message, instead of having to wait everytime